packages/shell-evaluator/src/shell-evaluator.ts

```diff
@@ -74,8 +74,49 @@ class ShellEvaluator {
             { original: removeCommand(input.trim()), rewritten: removeCommand(rewrittenInput.trim()) }
           );
         }
+
+        let Domain;
```

Comment:
Shell evaluator is not node / cli only. I don't have much context here, but would it be possible to place this in cli-repl?

Reply:
To me, this looks like it'd work as before for browser/java implementations, since we are `try`/`catch`ing domain on the line below.

Reply:
Yeah, it would work, but it’s also not a huge deal to move it … if cli-repl is the only user here that uses the Node.js core REPL implementation, it should be fine to only deal with this problem there